package.json

```diff
@@ -30,19 +30,19 @@
     "mongodb": "^3.1.13"
   },
   "devDependencies": {
-    "@types/cors": "^2.8.4",
-    "@types/dotenv": "^6.1.0",
-    "@types/express": "^4.16.1",
-    "@types/express-graphql": "^0.6.2",
-    "@types/graphql": "^14.0.7",
-    "@types/mongodb": "^3.1.21",
+    "@types/cors": "^2.8.12",
+    "@types/dotenv": "^8.2.0",
+    "@types/express": "^4.17.13",
+    "@types/express-graphql": "^0.9.0",
+    "@types/graphql": "^14.5.0",
+    "@types/mongodb": "^4.0.7",
```

Comment:
The types are now v 4, while the package itself `mongodb` is v 3, is not going to cause trouble?

Reply:
Good catch! Downgraded the types, but we should probably just upgrade mongo and remove the @types package, since the new mongo package comes with types. It has breaking changes though, so I will look into it during the day.

Reply:
@MargaretKrutikova Mongo is updated now. I've also removed the deprecated typing-packages.